### PR TITLE
[WIP] Cohesion time on page (PLATFORM-2414)

### PR DESCRIPTION
**Pull request by jo-rs**
Changed time on page events to fire using cohesion standards

### CODE REVIEW

File: src/desktop/analytics/main_layout.js [hunk 1]
```diff
@@ -102,13 +104,12 @@ class PageTimeTracker {
...
-
       window.analytics.track(
-        "Time on page",
-        {
-          category: this.description,
-          message: this.path,
-        },
+        ActionType.timeOnPage,
+        timeOnPage({
+          contextPageOwnerSlug: this.path.split("/")[2],
```
Comment (eessex) [inline src/desktop/analytics/main_layout.js:110]: This can use the `slug` and `pageType` defined above.

**Comment by eessex** [comment]
This project will need to be pushed to a review app for QA by the data-team cc @abhitip. See instructions here, you can open a new pr from a feature branch prefixed `review-app-`: https://github.com/artsy/force/blob/master/docs/creating_review_app.md#building-on-circle

**Comment by jo-rs** [comment]
@eessex propagation took a while, but: [Review App](https://new-time-on-page.artsy.net/)

**Comment by eessex** [comment]
closing in favor of https://github.com/artsy/force/pull/6023, using review app branch

**Pull request closed without merging**